cmd/juju/model/defaultscommand.go

```diff
@@ -45,20 +58,41 @@ See also:
 
 // NewDefaultsCommand wraps defaultsCommand with sane model settings.
 func NewDefaultsCommand() cmd.Command {
-	return modelcmd.WrapController(&defaultsCommand{})
+	defaultsCmd := &defaultsCommand{
+		newCloudAPI: func(caller base.APICallCloser) cloudAPI {
+			return cloudapi.NewClient(caller)
```

Comment:
Interesting, I have not seen this before...
More often, we get api root here directly from the command - this way there is less friction when newCloudAPI() is called.